### DISCUSSION TITLE
[Feature] backup to (shared) album

### DISCUSSION THREAD

**Discussion by ghost** [immich-app/immich#5310] [Feature Request]
### The feature

Hello maintainers of Immich,

I've recently started using your software and I'm missing the ability to backup folder X to a certain (shared) album. The same (old) feature request can be found [here](https://github.com/immich-app/immich/discussions/920), which was started just over a year ago now, and was 'planned in the future'. 

I request this feature because I want multiple users to be able to automatically backup a folder/album from a users mobile device to an album on Immich which is shared between them. 

I hope you will reconsider this feature request, and I thank you in advance.

Kind regards,
Squel.

### Platform

- [X] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
The feature request you linked is still open, so I'm going to close this as a duplicate. Also related: #1678
